### ISSUE TITLE
404 : Not Found. You are requesting a page that does not exist!

### ISSUE THREAD

**Issue by mpschr** [kirbs-/hide_code#68]
First off, kudos for the plugin! Should really go into the main repository. 

In any case, I installed the hide_code for my jupyter notebook and run into a problem when I want to export. I get redirected to a page which only says:

    404 : Not Found
    You are requesting a page that does not exist!

The installation was done using the `--user` flag without any kind of error message:

```
pip install hide_code
jupyter nbextension install --py hide_code --user
jupyter nbextension enable --py hide_code --user
jupyter serverextension enable --py hide_code --user
```

Firing up the notebook looks also fine to me:

> $ jupyter notebook --no-browser
> [I 14:03:58.764 NotebookApp] hide_code: Attempting to load hid_code export handler extensions.
> [I 14:03:58.766 NotebookApp] hide_code: Hide_code export handler extensions loaded.
> [I 14:03:58.779 NotebookApp] Loading IPython parallel extension
> [I 14:03:58.780 NotebookApp] Serving notebooks from local directory: /home/mpschr/Documents
> [I 14:03:58.780 NotebookApp] The Jupyter Notebook is running at:

And lastly, the versions:

> jupyter --version
> 4.4.0
> $ jupyter nbconvert --version
> 5.3.1
> $jupyter notebook --version
> 5.6.0
> $ python --version
> Python 3.6.6 :: Anaconda, Inc.

In summary - I really don't know how to tackle this one! Where should I look what is going wrong?

**Comment by kirbs-** [comment]
@mpschr Sorry for the delay! 404 errors are usually happen when the path to the notebook on the underlying OS is different than Jupyter is expecting. Post Jupyter's console output when the error occurs. It should show the full path of the file it's trying to open.

**Comment by kirbs-** [comment]
It's probably one of two things: 
1. export routes didn't load correctly. You can test this by restarting jupyter and going directly to ttp://localhost:8888/notebooks/export/latexpdf. 
2. Jupyter's notebook directory is different from the working directory it's launching from. To verify this one, do you see any logs showing the export started? e.g `hide_code: Starting Latex PDF export for ...`

**Comment by lordflaron** [comment]
Hi,
Sorry for the slow reply. 
1. Going directly to ```ttp://localhost:8888/notebooks/export/latexpdf``` gave me the 404 error.
2. It may have been an update that was pushed today, but the exporting actually works now, not sure what did it (both html -> pdf and latex -> pdf)

**Comment by kirbs-** [comment]
ok, i'll close this for now. Reopen if the problem resurfaces.

**Comment by lautaropacella** [comment]
I have this same issue and can't seem to solve it
